.github/workflows/test.yaml

```diff
@@ -57,5 +57,18 @@ jobs:
     - if: ${{ failure() }}
       uses: actions/upload-artifact@v3
       with:
-        name: cypress-screenshots-${{ runner.os }}-${{ matrix.node-version }}
+        name: cypress-screenshots-prod-${{ runner.os }}-${{ matrix.node-version }}
+        path: cypress/screenshots/
+
+    - run: npm run test:acceptance:dev
+      env:
+        CYPRESS_REQUEST_TIMEOUT: 20000
+        CYPRESS_DEFAULT_COMMAND_TIMEOUT: 40000
+        CYPRESS_PAGE_LOAD_TIMEOUT: 120000
+        CYPRESS_RETRIES: 3
+
+    - if: ${{ failure() }}
+      uses: actions/upload-artifact@v3
+      with:
+        name: cypress-screenshots-dev-${{ runner.os }}-${{ matrix.node-version }}
```

Comment:
I'm a bit worried about how long these tests are getting; I'm thinking we might need to parallelise them to get the run times down to something sensible.  Maybe we could look at what @colinrotherham is doing in https://github.com/alphagov/govuk-frontend/pull/2902 and try something similar.